todoapp/src/components/TodoForm.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import { defineProps, defineEmits, ref } from 'vue'
+
+const props = defineProps(['todos', 'inputContent'])
```

Comment:
Please add types to your emits and props definitions:
defineProps<{
  title?: string
  likes?: number
}>()

const emit = defineEmits<{
  (e: 'change', id: number): void
  (e: 'update', value: string): void
}>()